website/docs/d/nsxt_network_context_profile.html.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_network_context_profile"
+sidebar_current: "docs-vcd-data-source-nsxt-network-context-profile"
+description: |-
+  Provides a data source for NSX-T Network Context Profile lookup to later be used in Distributed
+  Firewall.
+---
+
+# vcd\_nsxt\_network\_context\_profile
+
+Provides a data source for NSX-T Network Context Profile lookup to later be used in Distributed
+Firewall.
+
+## Example Usage (SYSTEM scope network context profile lookup in a VDC Group)
+
+```hcl
+data "vcd_vdc_group" "existing" {
+  org  = "my-org" # Optional, can be inherited from Provider configuration
+  name = "main-vdc-group"
+}
+
+data "vcd_nsxt_network_context_profile" "cp1" {
+  context_id = data.vcd_vdc_group.existing.id
+  name       = "CTRXICA"
+  scope      = "SYSTEM"
+}
+```
+
+## Example Usage (SYSTEM profile lookup in an NSX-T Manager)
+```hcl
+data "vcd_nsxt_manager" "main" {
+  name = "first-nsxt-manager"
+}
+
+data "vcd_nsxt_network_context_profile" "p" {
+  context_id = data.vcd_nsxt_manager.main.id
+  name       = "CTRXICA"
+  scope      = "SYSTEM"
```

Comment:
Both examples are for SYSTEM. It may be helpful for someone if there was one more example for a non-SYSTEM scope.

Reply:
There is this problem, that we don't really have other types. Documentation exposes means for filtering, but as UI does not have a method to create them we're kind of limited here yet.